contentcuration/contentcuration/frontend/channelEdit/views/trash/TrashModal.vue

```diff
@@ -190,14 +180,19 @@
         return sortBy(this.getContentNodeChildren(this.trashId), 'modified').reverse();
       },
       backLink() {
-        const lastIndex = Math.max(0, this.$route.matched.length - 2);
```

Comment:
I am not sure what was the motivation behind

```
const lastIndex = Math.max(0, this.$route.matched.length - 2);
```

It seems that the trash modal can be currently visited only from the root tree view, or are there any other places from where it can be accessed?

Reply:
That was just to go back to wherever it came from, but yeah, can hardcode it to go to TREE_VIEW

Reply:
Okay, in case we need to navigate there from more places in the future, explicit routes should solve that hopefully (or full path if there is no other option), so I will leave like this. I hope that it will also self-document a bit from where this page can be accessed.